sim/druid/druid.go

```diff
@@ -66,6 +66,7 @@ type Druid struct {
 	TigersFury           *DruidSpell
 	Typhoon              *DruidSpell
 	Wrath                *DruidSpell
+	Thrash               *DruidSpell
```

Comment:
We've been adopting the convention of listing these in alphabetical order so that they're easier to find.